_posts/2022-04-10-using-log-collection.markdown

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "Log Collection on Flotta devices"
+date:   2022-04-10 12:00:00 +0100
+categories: flotta
+author: Eloy Coto <eloycoto@acalustra.com>
+tags:
+  - guide
+  - flotta
+  - logs
+
+summary-1: Introduction to flotta log collection
+---
+
+When running thousands of devices on the far edge, log collection retrieval
+maybe differs from one site to another, as also the limit of data that can be
+sent from the device or stored in the device.
+
+At the same time, workloads logs can vary in terms of importance, so the backup
+workload logs maybe need to be always pushed, and maybe another workload with
+less importance can lose logs on network connectivity issues. At the same time,
+maybe you don’t want that workload that manages any personal information sends
+the information outside the country.
+
+For that, Flotta introduces log collection targets, so each device - or
+deviceSets- will have a set of custom logs servers that one workload can use to
+send the logs.
+
+![Device Logs diagram](./assets/images/DeviceLogsConfig.png)
+
+So the main use case is to be able to define servers in edge devices and be
```

Comment:
"So the main use case is to be able to define servers in edge devices to be used by the workloads.". I didn't get what you had in mind in the other part of the sentence.